examples/Cassie/osc/standing_com_traj.cc

```diff
@@ -0,0 +1,90 @@
+#include "examples/Cassie/osc/standing_com_traj.h"
+
+#include <math.h>
+#include <string>
+#include "attic/multibody/rigidbody_utils.h"
+
+using std::cout;
+using std::endl;
+using std::string;
+
+using Eigen::MatrixXd;
+using Eigen::Vector2d;
+using Eigen::Vector3d;
+using Eigen::VectorXd;
+
+using dairlib::systems::OutputVector;
+using drake::systems::BasicVector;
+using drake::systems::Context;
+using drake::systems::LeafSystem;
+using drake::trajectories::PiecewisePolynomial;
+
+namespace dairlib {
+namespace cassie {
+namespace osc {
+
+StandingComTraj::StandingComTraj(const RigidBodyTree<double>& tree,
+                                 int pelvis_idx, int left_foot_idx,
+                                 int right_foot_idx, double height)
+    : tree_(tree),
+      pelvis_idx_(pelvis_idx),
+      left_foot_idx_(left_foot_idx),
+      right_foot_idx_(right_foot_idx),
+      height_(height) {
+  // Input/Output Setup
+  state_port_ = this
```

Comment:
Spacing looks off here

Reply:
Done. I'm not sure if my autoformatter is bugged...